src/main/resources/graphql/schema.graphqls

```diff
@@ -361,6 +362,8 @@ type GeofencingZoneProperties {
     start: Int
     end: Int
     rules: [GeofencingZoneRule]
+    "MultiPolygons where the lists of coordinates are encoded as polyline strings using precision of 5 decimals (see http://code.google.com/apis/maps/documentation/polylinealgorithm.html). Meant to be used instead of geometry.coordinates to minimize the response payload size."
+    polylineEncodedMultiPolygons: [[String]]
```

Comment:
I think it should be MultiPolygon singular?

Reply:
Does it make it easier to convert it back in the client?

Reply:
It might make it easier to understand the multipolygon format for those not already familiar with it, but shouldn't make any difference to how easy it will be to convert it back I think.